cockatrice/src/dlg_settings.cpp

```diff
@@ -875,8 +879,8 @@ void DlgSettings::retranslateUi()
     
     generalButton->setText(tr("General"));
     appearanceButton->setText(tr("Appearance"));
-    userInterfaceButton->setText(tr("User interface"));
-    deckEditorButton->setText(tr("Deck editor"));
+    userInterfaceButton->setText(tr("User Interface"));
+    deckEditorButton->setText(tr("Deck Editor"));
     messagesButton->setText(tr("Chat Settings"));
```

Comment:
if you're changing all this... do you mind changing
"Chat Settings" --> "Chat" as well

Chat Settings inside Settings... :)